lisa/nic.py

```diff
@@ -267,6 +268,18 @@ def reload(self) -> None:
         self.nics.clear()
         self._initialize()
 
+    @retry(tries=6, delay=20)
```

Comment:
Make the delay shorter and tries more. So it won't wait 20 seconds for next try.

Reply:
Changed so it checks with a backoff to retain the 120 second total wait and quick checks at the start with fewer total ssh roundtrips so we don't hog all the bandwidth while checking.

Reply:
1 second won't spend noticeable bandwidth, the only problem is it produces more log. But it saves up to 9 seconds for each run. How about change to at least 5 seconds?

Reply:
Oops, I thought I had pushed an update before adding that last comment.
Delay scheme, 15 tries, 3 seconds with 1.15x backoff 
calc_backoff(15,3,1.15)
3.4499999999999997
3.9674999999999994
4.562624999999999
5.247018749999998
6.034071562499997
6.939182296874995
7.980059641406244
9.17706858761718
10.553628875759756
12.136673207123719
13.957174188192276
16.050750316421116
18.458362863884282
21.22711729346692
24.411184887486957
max total wait: 167.15241747073347

My desire to add the backoff comes from testing where I saw concurrent tests running retries on nic.reload every second cause a bottleneck on the *orchestrator*, not the guest.

Reply:
Commit is updated now, sorry for confusion.